### PR TITLE
Create login page and auth

### PR DESCRIPTION
**Pull request by vasuchal**

### CODE REVIEW

File: backend/routes/index.js [hunk 1]
```diff
@@ -9,6 +10,14 @@ router.get("/", (req, res) => {
   res.send("Hello World!");
```
Comment (vasuchal) [inline backend/routes/index.js]: can delete this

File: backend/routes/index.js [hunk 2]
```diff
@@ -9,6 +10,14 @@ router.get("/", (req, res) => {
...
 });
 
+router.post('/:username', async (req, res) => {
```
Comment (vasuchal) [inline backend/routes/index.js]: remove :username

File: frontend/api/api.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import instance from '../src/axios';
+
+export const findOrCreateUser = async (patientInfo) => {
+    const requestString = `/users`;
+    console.log("HELLO")
```
Comment (vasuchal) [inline frontend/api/api.js]: remove console statements

File: frontend/src/App.jsx [hunk 4]
```diff
@@ -1,54 +1,23 @@
-import NavBar from "./components/NavBar.jsx"
-import Post from "./components/Post.jsx"
-import { useState } from "react"
-import ReactModal from "react-modal"
+
+import './App.css'
+import { BrowserRouter, Routes, Route, useLocation } from "react-router-dom";
+import { Home } from "./pages/Home"
+import './App.css'
```
Comment (exrlla) [inline frontend/src/App.jsx]: imported twice?

**Review by exrlla — APPROVED** [review]

**Pull request merged**